common/service/config/config.go

```diff
@@ -499,6 +501,18 @@ type (
 		// URI is the namespace default URI for visibility archiver
 		URI string `yaml:"URI"`
 	}
+
+	Authorization struct {
```

Comment:
JWTAuthorization ?

Reply:
I also plan to add mTLS authZ settings there. So it's not limited to JWT, but rather is meant to be used for all authZ settings